source/reference/array.rst

```diff
@@ -236,7 +236,200 @@ Unevaluated Items
 
 |draft2019-09|
 
-Documentation Coming Soon
+The ``unevaluatedItems`` keyword selects any data types not evaluated
+by an ``items``, ``prefixItems``, or `contains` keyword. Just as
+unevaluated"properties" affect only "properties" in an object, only
+"item"-related keywords affect unevaluated"items".
+
+It also applies inside valid subschemas with these keywords:
```

Comment:
Ideally here we'd just link to another document which explains "applicator keywords" rather than repeating this but ok there's probably no other place you can link to here right now so I suppose it's fine.